packages/data/src/world/mq/bottom_of_the_well_mq.yml

```diff
@@ -59,7 +41,108 @@
     "MQ Bottom of the Well Wonder Item Main Room Right 2": "can_use_slingshot"
     "MQ Bottom of the Well Wonder Item Main Room Right 3": "can_use_slingshot"
     "MQ Bottom of the Well Wonder Item Main Room Right 4": "can_use_slingshot"
-    "MQ Bottom of the Well Wonder Item Side Room 1": "(can_play_zelda || (has_explosives_or_hammer && has_hover_boots)) && has_lens && can_use_slingshot"
-    "MQ Bottom of the Well Wonder Item Side Room 2": "(can_play_zelda || (has_explosives_or_hammer && has_hover_boots)) && has_lens && can_use_slingshot"
-    "MQ Bottom of the Well Wonder Item Side Room 3": "(can_play_zelda || (has_explosives_or_hammer && has_hover_boots)) && has_lens && can_use_slingshot"
-    "MQ Bottom of the Well Wonder Item Side Room 4": "(can_play_zelda || (has_explosives_or_hammer && has_hover_boots)) && has_lens && can_use_slingshot"
+"Bottom of the Well Main Room Center Switch Corner":
+  dungeon: BotW
+  exits:
+    "Bottom of the Well Main": "has_explosives_or_hammer"
```

Comment:
Can surely hook or climb anywhere out of here.

Reply:
climb does work; fixed.

Reply:
Hook works. Add it.

Reply:
Added.